release_dashboard/lib/enums/cherrypick.dart

```diff
@@ -0,0 +1,9 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// Keys for engine/framework cherrypicks which are in [Map<K, V>] format.
```

Comment:
engine/framework -> repository

Reply:
Even though I was the one who recommended making this an enum, I completely forget the context of what this is, and the comment does not help. This is completely for the purpose of a single widget, right? If so, I would recommend defining this enum in the same file with that widget, and then giving a more specific dartdoc as to its purpose.

Reply:
The enum is used across different widgets. There are several widgets that read the cherrypick map, and they use the same enum to query the values. That was why it is placed in a common enums folder (now models). 

I added a more specific dartdoc to explain the enums. Please take another look, and let me know if it works.